.github/workflows/ubuntu-cminpack-install.yaml

```diff
@@ -41,11 +36,6 @@ jobs:
       
       - name: Test cminpack
         run: ctest --test-dir ${{ env.CMINPACK_BUILD_DIR }} --build-config ${{ matrix.CMAKE_BUILD_TYPE }}
-        if: ${{ matrix.USE_BLAS == 'OFF' }}
-      
-      - name: Test cminpack with BLAS
-        run: ctest --test-dir ${{ env.CMINPACK_BUILD_DIR }} --build-config ${{ matrix.CMAKE_BUILD_TYPE }} --exclude-regex tlmdifc
```

Comment:
In regards of BLAS comments, to be honest, I don't have any experience with it and I'm unable to say anything about each implementation. I think it should not be too hard to rewrite such workflows logic to test for OpenBLAS.

When I saw this exclude regex in the tests, it sounded strange to me, and I thought it was a library-specific test fail such that other libraries could solve. I mean, when I'm learning how to build / test a library, I often check their workflows, in case they exist. The testing for openblas with such regex is not natural for users. Most CMake users expect a ```ctest --test-dir <dir> -C <config>``` as default testing for a build configuration.

In my view, such exclude ```tlmdifc``` pattern for OpenBLAS should be inside the test manager file [https://github.com/devernay/cminpack/blob/master/examples/CMakeLists.txt](https://github.com/devernay/cminpack/blob/master/examples/CMakeLists.txt), hidden from the users expectation.

Reply:
> I thought it was a library-specific test fail such that other libraries could solve.

No, it's just that the way unit tests are written in this package is not the best one. See:
- https://github.com/cbouilla/minpack-1.1?tab=readme-ov-file#testing-minpack
- https://github.com/devernay/cminpack/issues/37 
- https://github.com/cbouilla/minpack-1.1/issues/1